sql2.txt

```diff
@@ -3,12 +3,22 @@ to join the OrderDetails table with the product table.  For TotalCount, you will
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
 
+SELECT ProductName, SUM(od.Quantity) AS TotalCount 
+FROM orders AS o 
+JOIN orderDetails AS od ON o.orderID=od.orderID 
+JOIN products AS p ON od.ProductID=p.ProductID 
+GROUP BY p.ProductID 
+ORDER BY p.ProductName;
 
 2. Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
 ordered by customer name. Paste your SQL statement below. You will have to join the Customers
 table with the Orders table, and the Orders table with the OrderDetails table, 
 because the Orders table is the one that ties the Customer table to the OrderDetails table.
 
+SELECT CustomerName, OrderDetailID 
+FROM Customers AS c 
+JOIN  orders AS o  ON c.CustomerID=o.CustomerID 
+JOIN orderDetails AS od ON o.orderID=od.orderID;
```

Comment:
The instructions require this answer to be "ordered by customer name", but based on your answer to # 1 above, I think you already know how to do that and simply missed this part of the instructions. If you actually don't know how to order this by customer name, let me know.

Reply:
Done